app/assets/javascripts/startpage/surveyQuestions.coffee

```diff
@@ -7,19 +7,19 @@ surveyQuestions.config(["$routeProvider", ($routeProvider) ->
         templateUrl: "/assets/survey_questions.html",
         controller: "SurveyQuestionsController"
     })
+    .when("/questions/:id/edit", {
+        templateUrl: "/assets/survey_questions_edit.html",
+        controller: "SurveyQuestionsController"
+    })
```

Comment:
Maybe if you can pass in args or use state knowledge features to reuse the normal sureyquestions partial.